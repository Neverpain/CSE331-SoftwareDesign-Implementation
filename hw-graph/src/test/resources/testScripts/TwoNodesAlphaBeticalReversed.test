# Tests adding two nodes to a graph, and a pair of nodes (in reverse order) will
# print in alphabetical order

# Create a graph
CreateGraph graph1

# Add two nodes
AddedNode graph1 n2
AddedNode graph1 n1

# Print the nodes contained in the graph and outgoing edges to the nodes
graph1 contains: n1 n2
the children of n1 in graph1 are:
the children of n2 in graph1 are: