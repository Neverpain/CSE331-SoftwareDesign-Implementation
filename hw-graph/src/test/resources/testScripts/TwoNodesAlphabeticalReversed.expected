# Tests adding two nodes to a graph, and a pair of nodes (in reverse order) will
# print in alphabetical order

# Create a graph
created graph graph1

# Add two nodes
added node n2 to graph1
added node n1 to graph1

# Print the nodes contained in the graph and outgoing edges to the nodes
graph1 contains: n1 n2
the children of n1 in graph1 are:
the children of n2 in graph1 are: